src/WebJobs.Script/Metrics/MetricConstants.cs

```diff
@@ -0,0 +1,28 @@
+// Copyright (c) .NET Foundation. All rights reserved.
+// Licensed under the MIT License. See License.txt in the project root for license information.
+
+namespace Microsoft.Azure.WebJobs.Script.Metrics
+{
+    public class MetricConstants
+    {
+        // Cloud Information
+        public const string CloudPlatformName = "azure_functions";
+
+        // Meters
+        public const string AzureFunctions = "azure.functions";
```

Comment:
Meter names are intended to be fully quality namespaces.

Something like `Microsoft.Azure.WebJobs.Script.Host.Internal` would be preferred. Note, I added `.Internal` here because I want to express customers/extensions are not to take a dependency on this meter, and we reserve the right to perform breaking changes on it with no notice.

We can have official customer-facing ones as part of the broader OTel work.